homeassistant/components/scsgate.py

```diff
@@ -0,0 +1,154 @@
+"""
+homeassistant.components.scsgate
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Provides support for SCSGate components.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/scsgate/
+"""
+import logging
+from threading import Lock
+from homeassistant.core import EVENT_HOMEASSISTANT_STOP
+
+from scsgate.connection import Connection
+from scsgate.reactor import Reactor
+from scsgate.tasks import GetStatusTask
+from scsgate.messages import StateMessage, ScenarioTriggeredMessage
+
+DOMAIN = "scsgate"
+
```

Comment:
This file should declare a `REQUIREMENTS` list with dependencies that you expect to be installed before running this component. After that, run `script/gen_requirements_all.py` to update the `requirements_all.txt` file.